extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -422,10 +422,10 @@ function tests(name) {
     it('should add container info when ad has a container', () => {
       return getAdInAdContainer().then(obj => {
         const ampAd = obj.ampAd;
+        const impl = ampAd.implementation_;
         expect(ampAd.getAttribute('amp-container-element')).to.be.null;
-        ampAd.implementation_.onLayoutMeasure();
-        expect(ampAd.getAttribute('amp-container-element'))
-            .to.equal('AMP-STICKY-AD');
+        impl.onLayoutMeasure();
+        expect(impl.container_.tagName).to.equal('AMP-STICKY-AD');
```

Comment:
As we discussed offline, I will not stub a top level import function here. Instead will wait `layoutCallback` resolve, and check created iframe's url.